torch/_tensor_docs.py

```diff
@@ -4681,3 +4686,35 @@ def callable(a, b) -> number
 in the output mirror changes in ``self``, and the output stays attached
 to the autograd graph. ``cls`` must be a subclass of ``Tensor``.
 """)
+
+add_docstr_all('crow_indices',
+               r"""
+crow_indices() -> IntTensor
+
+Returns the tensor containing the compressed row indices of the :attr:`self`
+tensor when :attr:`self` is a sparse CSR tensor of layout ``sparse_csr``.
+The ``crow_indices`` tensor is strictly of type ``int32`` and of shape
```

Comment:
Needs an update: "strictly of type ``int32``" -> "of type ``int32`` or ``int64``".